Source/Utils/CoarsenIO.H

```diff
@@ -135,6 +135,13 @@ namespace CoarsenIO{
                    const int ncomp,
                    const int ngrow,
                    const IntVect crse_ratio=IntVect(1) );
+    void Coarsen ( MultiFab& mf_dst,
+                   const MultiFab& mf_src,
+                   const int dcomp,
+                   const int scomp,
+                   const int ncomp,
+                   const IntVect ngrowvect,
+                   const IntVect crse_ratio=IntVect(1) );
```

Comment:
Could you explain why we need 2 versions of `Coarsen` here?

Reply:
My laziness :) There are many calls to the Coarsen routine in the in the diagnostic code that call it passing in an integer for ngrow. Rather than changing all of them, I added the extra routine with the different argument list.

Reply:
OK, sounds good. Does it mean that there should be a second PR to fix the remain calls to `Coarsen` with only scalar guard cells?

Reply:
@RemiLehe If you think it would be cleaner to change the calls to Coarsen to pass in an IntVect, I can make the change in a new PR.